tests/data_context/test_data_context.py

```diff
@@ -438,12 +444,12 @@ def test_data_context_profile_datasource_on_non_existent_one_raises_helpful_erro
 
 @freeze_time("09/26/2019 13:42:41")
 @pytest.mark.rendered_output
-def test_render_full_static_site_from_empty_project(tmp_path_factory, filesystem_csv_3):
+def test_render_full_static_site_from_empty_project(tmpdir, filesystem_csv_3):
```

Comment:
@Aylr is this a good use case for `tmp_path` as well?

Reply:
Most definitely!

Reply:
Fixed.